### DISCUSSION TITLE
import node_modules into scss

### DISCUSSION THREAD

**Discussion by premier213** [vercel/turborepo#1730] [Help] [answered]
Hi everyone, when I import the Tilda address for import from node_modules I get an error
how to access root nodE_modules without a relative path?

`Error: Failed to find '~antd/dist/antd.css'
  in [
    /home/***/packages/UI/src/app/styles/vendors
  ]
    at /home/***turborepo/node_modules/postcss-import/lib/resolve-id.js:35:13
    at async Promise.all (index 0)
    at async Promise.all (index 0)
    at async LazyResult.runAsync (/home/premier/projects/zarin/zarin-turborepo/node_modules/postcss/lib/lazy-result.js:396:11)

Node.js v18.7.0`

**Comment by anthonyshew** [comment, marked as answer]
Closing due to staleness and not looking related to Turborepo.
